lib/admin.php

```diff
@@ -54,6 +54,7 @@ function __construct() {
 				'pinterest'          => 1,
 				'linkedin'           => 1,
 				'stumbleupon'        => 1,
+				'love'        		 => 1,
```

Comment:
looks like there is a space vs/ tabs issue here. 

The general rule of thumb is use actual tabs before the first character of a line to indent. After the first character use spaces, not tabs, to line up the lines. This will allow the lines to match no matter what editor you are using. The difference is amplified by github so you can see the issue clearly above.